app/src/main/java/com/locoquest/app/service/BenchmarkService.kt

```diff
@@ -21,14 +22,14 @@ import retrofit2.Response
 interface IBenchmarkService {
 }
 
-open class BenchmarkService(application:Application){
+open class BenchmarkService(application:Application) {
+
     private val application = application
 
-    open fun parseBenchmarkData(benchmarkJson: String) {
+    open suspend fun parseBenchmarkData(benchmarkJson: String) {
         withContext(Dispatchers.IO) {
             val service = RetrofitClientInstance.retrofitInstance?.create(IBenchmarkDAO::class.java)
+            // perform any IO operation here
```

Comment:
Made a few changes here to allow it to compile. Used suggestions from the IDE to allow this to work

Reply:
Don't put conversation in source code comments.  That's clutter.  Put the conversation here, in the Files Changed section of the pull request, where you can have a dialog with other developers.